Shared/Actions/ExplicitResult.cs

```diff
@@ -43,6 +43,79 @@ public class ExplicitResult : RawUploadResult
         [DataMember(Name = "quality_analysis")]
         public QualityAnalysis QualityAnalysis { get; protected set; }
 
+        /// <summary>
+        /// Color information: predominant colors and histogram of 32 leading colors.
+        /// </summary>
+        [DataMember(Name = "colors")]
+        public string[][] Colors { get; protected set; }
+
+        /// <summary>
+        /// IPTC, XMP, and detailed Exif metadata. Supported for images, video, and audio.
+        /// </summary>
+        [DataMember(Name = "image_metadata")]
+        public Dictionary<string, string> Metadata { get; protected set; }
```

Comment:
To avoid ambiguity with structured metadata, let's call it image metadata. I know it is already called `Metadata` in other places, that needs to be changed to `ImageMetadata` and deprecated as well.